Raspberry Pi support: no

Hi all,

I'm using a Raspberry Pi 2 Model B Rev 1.1 - For those who don't know how, I found it using the code below:

 `dmesg | grep 'Machine model'`

I have been trying to get the libcec working for the last two days, and I haven't had much luck.  I've tried following the instructions located in the [read me](https://github.com/Pulse-Eight/libcec/blob/master/README.md) file, but those didn't work.  I wound up being unable to connect to the TV. 

I have tried a couple of the ways in [Complete installation instructions on Raspbian #84](https://github.com/Pulse-Eight/libcec/issues/new), and that had the same issue I had in the first attempt.

I've tried to dig around to figure this out, but I'm not really sure what is happening.  I have a feeling that it is related to the line that says `Raspberry Pi support: no` that comes up after the command `cmake -DRPI_INCLUDE_DIR=/opt/vc/include -DRPI_LIB_DIR=/opt/vc/lib ..`.

Here is what I am doing:
```
pi@raspberrypi ~ $ sudo apt-get update
Get:1 http://archive.raspberrypi.org jessie InRelease [13.3 kB]
Get:2 http://mirrordirector.raspbian.org jessie InRelease [15.0 kB]
Get:3 http://archive.raspberrypi.org jessie/main Sources [31.5 kB]
Get:4 http://mirrordirector.raspbian.org jessie/main armhf Packages [8,962 kB]
Get:5 http://archive.raspberrypi.org jessie/ui Sources [5,197 B]
Get:6 http://archive.raspberrypi.org jessie/main armhf Packages [101 kB]
Get:7 http://archive.raspberrypi.org jessie/ui armhf Packages [7,639 B]
Ign http://archive.raspberrypi.org jessie/main Translation-en_GB
Ign http://archive.raspberrypi.org jessie/main Translation-en
Ign http://archive.raspberrypi.org jessie/ui Translation-en_GB
Ign http://archive.raspberrypi.org jessie/ui Translation-en
Get:8 http://mirrordirector.raspbian.org jessie/contrib armhf Packages [37.5 kB]
Get:9 http://mirrordirector.raspbian.org jessie/non-free armhf Packages [70.2 kB]
Get:10 http://mirrordirector.raspbian.org jessie/rpi armhf Packages [1,356 B]
Ign http://mirrordirector.raspbian.org jessie/contrib Translation-en_GB
Ign http://mirrordirector.raspbian.org jessie/contrib Translation-en
Ign http://mirrordirector.raspbian.org jessie/main Translation-en_GB
Ign http://mirrordirector.raspbian.org jessie/main Translation-en
Ign http://mirrordirector.raspbian.org jessie/non-free Translation-en_GB
Ign http://mirrordirector.raspbian.org jessie/non-free Translation-en
Ign http://mirrordirector.raspbian.org jessie/rpi Translation-en_GB
Ign http://mirrordirector.raspbian.org jessie/rpi Translation-en
Fetched 9,244 kB in 36s (254 kB/s)
Reading package lists... Done
pi@raspberrypi ~ $ apt-get install cmake liblockdev1-dev libudev-dev libxrandr-dev python-dev swig
E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?
pi@raspberrypi ~ $ sudo apt-get install cmake liblockdev1-dev libudev-dev libxrandr-dev python-dev swig
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following extra packages will be installed:
  cmake-data liblockdev1 libpthread-stubs0-dev libpython-dev libpython2.7-dev libx11-dev libx11-doc libxau-dev libxcb1-dev libxdmcp-dev libxext-dev libxrender-dev
  python2.7-dev swig2.0 x11proto-core-dev x11proto-input-dev x11proto-kb-dev x11proto-randr-dev x11proto-render-dev x11proto-xext-dev xorg-sgml-doctools xtrans-dev
Suggested packages:
  codeblocks eclipse ninja-build libxcb-doc libxext-doc swig-doc swig-examples swig2.0-examples swig2.0-doc
The following NEW packages will be installed:
  cmake cmake-data liblockdev1 liblockdev1-dev libpthread-stubs0-dev libpython-dev libpython2.7-dev libudev-dev libx11-dev libx11-doc libxau-dev libxcb1-dev libxdmcp-dev
  libxext-dev libxrandr-dev libxrender-dev python-dev python2.7-dev swig swig2.0 x11proto-core-dev x11proto-input-dev x11proto-kb-dev x11proto-randr-dev x11proto-render-dev
  x11proto-xext-dev xorg-sgml-doctools xtrans-dev
0 upgraded, 28 newly installed, 0 to remove and 70 not upgraded.
Need to get 27.1 MB of archives.
After this operation, 62.9 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://mirrordirector.raspbian.org/raspbian/ jessie/main python-dev armhf 2.7.9-1 [1,188 B]
Get:2 http://mirrordirector.raspbian.org/raspbian/ jessie/main cmake-data all 3.0.2-1 [929 kB]
Get:3 http://mirrordirector.raspbian.org/raspbian/ jessie/main cmake armhf 3.0.2-1 [1,870 kB]
Get:4 http://mirrordirector.raspbian.org/raspbian/ jessie/main libpython2.7-dev armhf 2.7.9-2 [17.9 MB]
Get:5 http://mirrordirector.raspbian.org/raspbian/ jessie/main liblockdev1 armhf 1.0.3-1.6+b1 [12.7 kB]
Get:6 http://mirrordirector.raspbian.org/raspbian/ jessie/main liblockdev1-dev armhf 1.0.3-1.6+b1 [7,804 B]
Get:7 http://mirrordirector.raspbian.org/raspbian/ jessie/main libpthread-stubs0-dev armhf 0.3-4 [4,042 B]
Get:8 http://mirrordirector.raspbian.org/raspbian/ jessie/main libpython-dev armhf 2.7.9-1 [19.6 kB]
Get:9 http://mirrordirector.raspbian.org/raspbian/ jessie/main libudev-dev armhf 215-17+deb8u2 [23.2 kB]
Get:10 http://mirrordirector.raspbian.org/raspbian/ jessie/main xorg-sgml-doctools all 1:1.11-1 [21.9 kB]
Get:11 http://mirrordirector.raspbian.org/raspbian/ jessie/main x11proto-core-dev all 7.0.26-1 [729 kB]
Get:12 http://mirrordirector.raspbian.org/raspbian/ jessie/main libxau-dev armhf 1:1.0.8-1 [23.0 kB]
Get:13 http://mirrordirector.raspbian.org/raspbian/ jessie/main libxdmcp-dev armhf 1:1.1.1-1 [40.8 kB]
Get:14 http://mirrordirector.raspbian.org/raspbian/ jessie/main x11proto-input-dev all 2.3.1-1 [157 kB]
Get:15 http://mirrordirector.raspbian.org/raspbian/ jessie/main x11proto-kb-dev all 1.0.6-2 [269 kB]
Get:16 http://mirrordirector.raspbian.org/raspbian/ jessie/main xtrans-dev all 1.3.4-1 [99.3 kB]
Get:17 http://mirrordirector.raspbian.org/raspbian/ jessie/main libxcb1-dev armhf 1.10-3 [78.1 kB]
Get:18 http://mirrordirector.raspbian.org/raspbian/ jessie/main libx11-dev armhf 2:1.6.2-3 [726 kB]
Get:19 http://mirrordirector.raspbian.org/raspbian/ jessie/main libx11-doc all 2:1.6.2-3 [2,026 kB]
Get:20 http://mirrordirector.raspbian.org/raspbian/ jessie/main x11proto-xext-dev all 7.3.0-1 [212 kB]
Get:21 http://mirrordirector.raspbian.org/raspbian/ jessie/main libxext-dev armhf 2:1.3.3-1 [102 kB]
Get:22 http://mirrordirector.raspbian.org/raspbian/ jessie/main x11proto-randr-dev all 1.4.0-2 [46.8 kB]
Get:23 http://mirrordirector.raspbian.org/raspbian/ jessie/main x11proto-render-dev all 2:0.11.1-2 [20.8 kB]
Get:24 http://mirrordirector.raspbian.org/raspbian/ jessie/main libxrender-dev armhf 1:0.9.8-1 [38.8 kB]
Get:25 http://mirrordirector.raspbian.org/raspbian/ jessie/main libxrandr-dev armhf 2:1.4.2-1 [39.3 kB]
Get:26 http://mirrordirector.raspbian.org/raspbian/ jessie/main python2.7-dev armhf 2.7.9-2 [281 kB]
Get:27 http://mirrordirector.raspbian.org/raspbian/ jessie/main swig2.0 armhf 2.0.12-1 [1,071 kB]
Get:28 http://mirrordirector.raspbian.org/raspbian/ jessie/main swig armhf 2.0.12-1 [285 kB]
Fetched 27.1 MB in 1min 6s (407 kB/s)
Selecting previously unselected package cmake-data.
(Reading database ... 116795 files and directories currently installed.)
Preparing to unpack .../cmake-data_3.0.2-1_all.deb ...
Unpacking cmake-data (3.0.2-1) ...
Selecting previously unselected package cmake.
Preparing to unpack .../cmake_3.0.2-1_armhf.deb ...
Unpacking cmake (3.0.2-1) ...
Selecting previously unselected package libpython2.7-dev:armhf.
Preparing to unpack .../libpython2.7-dev_2.7.9-2_armhf.deb ...
Unpacking libpython2.7-dev:armhf (2.7.9-2) ...
Selecting previously unselected package liblockdev1.
Preparing to unpack .../liblockdev1_1.0.3-1.6+b1_armhf.deb ...
Unpacking liblockdev1 (1.0.3-1.6+b1) ...
Selecting previously unselected package liblockdev1-dev.
Preparing to unpack .../liblockdev1-dev_1.0.3-1.6+b1_armhf.deb ...
Unpacking liblockdev1-dev (1.0.3-1.6+b1) ...
Selecting previously unselected package libpthread-stubs0-dev:armhf.
Preparing to unpack .../libpthread-stubs0-dev_0.3-4_armhf.deb ...
Unpacking libpthread-stubs0-dev:armhf (0.3-4) ...
Selecting previously unselected package libpython-dev:armhf.
Preparing to unpack .../libpython-dev_2.7.9-1_armhf.deb ...
Unpacking libpython-dev:armhf (2.7.9-1) ...
Selecting previously unselected package libudev-dev:armhf.
Preparing to unpack .../libudev-dev_215-17+deb8u2_armhf.deb ...
Unpacking libudev-dev:armhf (215-17+deb8u2) ...
Selecting previously unselected package xorg-sgml-doctools.
Preparing to unpack .../xorg-sgml-doctools_1%3a1.11-1_all.deb ...
Unpacking xorg-sgml-doctools (1:1.11-1) ...
Selecting previously unselected package x11proto-core-dev.
Preparing to unpack .../x11proto-core-dev_7.0.26-1_all.deb ...
Unpacking x11proto-core-dev (7.0.26-1) ...
Selecting previously unselected package libxau-dev:armhf.
Preparing to unpack .../libxau-dev_1%3a1.0.8-1_armhf.deb ...
Unpacking libxau-dev:armhf (1:1.0.8-1) ...
Selecting previously unselected package libxdmcp-dev:armhf.
Preparing to unpack .../libxdmcp-dev_1%3a1.1.1-1_armhf.deb ...
Unpacking libxdmcp-dev:armhf (1:1.1.1-1) ...
Selecting previously unselected package x11proto-input-dev.
Preparing to unpack .../x11proto-input-dev_2.3.1-1_all.deb ...
Unpacking x11proto-input-dev (2.3.1-1) ...
Selecting previously unselected package x11proto-kb-dev.
Preparing to unpack .../x11proto-kb-dev_1.0.6-2_all.deb ...
Unpacking x11proto-kb-dev (1.0.6-2) ...
Selecting previously unselected package xtrans-dev.
Preparing to unpack .../xtrans-dev_1.3.4-1_all.deb ...
Unpacking xtrans-dev (1.3.4-1) ...
Selecting previously unselected package libxcb1-dev:armhf.
Preparing to unpack .../libxcb1-dev_1.10-3_armhf.deb ...
Unpacking libxcb1-dev:armhf (1.10-3) ...
Selecting previously unselected package libx11-dev:armhf.
Preparing to unpack .../libx11-dev_2%3a1.6.2-3_armhf.deb ...
Unpacking libx11-dev:armhf (2:1.6.2-3) ...
Selecting previously unselected package libx11-doc.
Preparing to unpack .../libx11-doc_2%3a1.6.2-3_all.deb ...
Unpacking libx11-doc (2:1.6.2-3) ...
Selecting previously unselected package x11proto-xext-dev.
Preparing to unpack .../x11proto-xext-dev_7.3.0-1_all.deb ...
Unpacking x11proto-xext-dev (7.3.0-1) ...
Selecting previously unselected package libxext-dev:armhf.
Preparing to unpack .../libxext-dev_2%3a1.3.3-1_armhf.deb ...
Unpacking libxext-dev:armhf (2:1.3.3-1) ...
Selecting previously unselected package x11proto-randr-dev.
Preparing to unpack .../x11proto-randr-dev_1.4.0-2_all.deb ...
Unpacking x11proto-randr-dev (1.4.0-2) ...
Selecting previously unselected package x11proto-render-dev.
Preparing to unpack .../x11proto-render-dev_2%3a0.11.1-2_all.deb ...
Unpacking x11proto-render-dev (2:0.11.1-2) ...
Selecting previously unselected package libxrender-dev:armhf.
Preparing to unpack .../libxrender-dev_1%3a0.9.8-1_armhf.deb ...
Unpacking libxrender-dev:armhf (1:0.9.8-1) ...
Selecting previously unselected package libxrandr-dev:armhf.
Preparing to unpack .../libxrandr-dev_2%3a1.4.2-1_armhf.deb ...
Unpacking libxrandr-dev:armhf (2:1.4.2-1) ...
Selecting previously unselected package python2.7-dev.
Preparing to unpack .../python2.7-dev_2.7.9-2_armhf.deb ...
Unpacking python2.7-dev (2.7.9-2) ...
Selecting previously unselected package python-dev.
Preparing to unpack .../python-dev_2.7.9-1_armhf.deb ...
Unpacking python-dev (2.7.9-1) ...
Selecting previously unselected package swig2.0.
Preparing to unpack .../swig2.0_2.0.12-1_armhf.deb ...
Unpacking swig2.0 (2.0.12-1) ...
Selecting previously unselected package swig.
Preparing to unpack .../swig_2.0.12-1_armhf.deb ...
Unpacking swig (2.0.12-1) ...
Processing triggers for man-db (2.7.0.2-5) ...
Setting up cmake-data (3.0.2-1) ...
Setting up cmake (3.0.2-1) ...
Setting up libpython2.7-dev:armhf (2.7.9-2) ...
Setting up liblockdev1 (1.0.3-1.6+b1) ...
Setting up liblockdev1-dev (1.0.3-1.6+b1) ...
Setting up libpthread-stubs0-dev:armhf (0.3-4) ...
Setting up libpython-dev:armhf (2.7.9-1) ...
Setting up libudev-dev:armhf (215-17+deb8u2) ...
Setting up xorg-sgml-doctools (1:1.11-1) ...
Setting up x11proto-core-dev (7.0.26-1) ...
Setting up libxau-dev:armhf (1:1.0.8-1) ...
Setting up libxdmcp-dev:armhf (1:1.1.1-1) ...
Setting up x11proto-input-dev (2.3.1-1) ...
Setting up x11proto-kb-dev (1.0.6-2) ...
Setting up xtrans-dev (1.3.4-1) ...
Setting up libxcb1-dev:armhf (1.10-3) ...
Setting up libx11-dev:armhf (2:1.6.2-3) ...
Setting up libx11-doc (2:1.6.2-3) ...
Setting up x11proto-xext-dev (7.3.0-1) ...
Setting up libxext-dev:armhf (2:1.3.3-1) ...
Setting up x11proto-randr-dev (1.4.0-2) ...
Setting up x11proto-render-dev (2:0.11.1-2) ...
Setting up libxrender-dev:armhf (1:0.9.8-1) ...
Setting up libxrandr-dev:armhf (2:1.4.2-1) ...
Setting up python2.7-dev (2.7.9-2) ...
Setting up python-dev (2.7.9-1) ...
Setting up swig2.0 (2.0.12-1) ...
Setting up swig (2.0.12-1) ...
Processing triggers for libc-bin (2.19-18+deb8u1) ...
pi@raspberrypi ~ $ mkdir platform/build
mkdir: cannot create directory ‘platform/build’: No such file or directory
pi@raspberrypi ~ $ sudo mkdir platform/build
mkdir: cannot create directory ‘platform/build’: No such file or directory
pi@raspberrypi ~ $ git clone https://github.com/Pulse-Eight/platform.git
Cloning into 'platform'...
remote: Counting objects: 449, done.
remote: Total 449 (delta 0), reused 0 (delta 0), pack-reused 449
Receiving objects: 100% (449/449), 156.58 KiB | 0 bytes/s, done.
Resolving deltas: 100% (257/257), done.
Checking connectivity... done.
pi@raspberrypi ~ $ mkdir platform/build
pi@raspberrypi ~ $ cd platform/build
pi@raspberrypi ~/platform/build $ cmake ..
-- The C compiler identification is GNU 4.9.2
-- The CXX compiler identification is GNU 4.9.2
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Looking for include file pthread.h
-- Looking for include file pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Looking for __atomic_fetch_add_4 in atomic
-- Looking for __atomic_fetch_add_4 in atomic - found
-- Performing Test HAS_BUILTIN_SYNC_ADD_AND_FETCH
-- Performing Test HAS_BUILTIN_SYNC_ADD_AND_FETCH - Success
-- Performing Test HAS_BUILTIN_SYNC_SUB_AND_FETCH
-- Performing Test HAS_BUILTIN_SYNC_SUB_AND_FETCH - Success
-- Performing Test HAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP
-- Performing Test HAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP - Success
-- Configuring done
-- Generating done
-- Build files have been written to: /home/pi/platform/build
pi@raspberrypi ~/platform/build $ make
Scanning dependencies of target p8-platform
[100%] Building CXX object CMakeFiles/p8-platform.dir/src/util/StringUtils.cpp.o
Linking CXX static library libp8-platform.a
[100%] Built target p8-platform
pi@raspberrypi ~/platform/build $ make install
[100%] Built target p8-platform
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/lib/libp8-platform.a
CMake Error at cmake_install.cmake:36 (file):
  file INSTALL cannot copy file "/home/pi/platform/build/libp8-platform.a" to
  "/usr/local/lib/libp8-platform.a".


Makefile:66: recipe for target 'install' failed
make: *** [install] Error 1
pi@raspberrypi ~/platform/build $ sudo make install
[100%] Built target p8-platform
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/lib/libp8-platform.a
-- Installing: /usr/local/include/p8-platform/os.h
-- Installing: /usr/local/include/p8-platform/posix/os-socket.h
-- Installing: /usr/local/include/p8-platform/posix/os-threads.h
-- Installing: /usr/local/include/p8-platform/posix/os-types.h
-- Installing: /usr/local/include/p8-platform/sockets/cdevsocket.h
-- Installing: /usr/local/include/p8-platform/sockets/socket.h
-- Installing: /usr/local/include/p8-platform/sockets/tcp.h
-- Installing: /usr/local/include/p8-platform/threads/atomics.h
-- Installing: /usr/local/include/p8-platform/threads/mutex.h
-- Installing: /usr/local/include/p8-platform/threads/threads.h
-- Installing: /usr/local/include/p8-platform/util/atomic.h
-- Installing: /usr/local/include/p8-platform/util/buffer.h
-- Installing: /usr/local/include/p8-platform/util/StringUtils.h
-- Installing: /usr/local/include/p8-platform/util/StdString.h
-- Installing: /usr/local/include/p8-platform/util/timeutils.h
-- Installing: /usr/local/include/p8-platform/util/util.h
-- Installing: /usr/local/lib/pkgconfig/p8-platform.pc
-- Installing: /usr/local/lib/p8-platform/p8-platform-config.cmake
pi@raspberrypi ~/platform/build $ cd
pi@raspberrypi ~ $ git clone https://github.com/Pulse-Eight/libcec.git
Cloning into 'libcec'...
remote: Counting objects: 13910, done.
remote: Total 13910 (delta 0), reused 0 (delta 0), pack-reused 13910
Receiving objects: 100% (13910/13910), 4.86 MiB | 1.30 MiB/s, done.
Resolving deltas: 100% (10604/10604), done.
Checking connectivity... done.
pi@raspberrypi ~ $ mkdir libcec/build
pi@raspberrypi ~ $ cd libcec/build
pi@raspberrypi ~/libcec/build $ cmake ..
-- The C compiler identification is GNU 4.9.2
-- The CXX compiler identification is GNU 4.9.2
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Performing Test SUPPORTS_CXX11
-- Performing Test SUPPORTS_CXX11 - Success
-- Looking for include file pthread.h
-- Looking for include file pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Looking for initscr in curses
-- Looking for initscr in curses - not found
-- Looking for dlopen in dl
-- Looking for dlopen in dl - found
-- Looking for clock_gettime in rt
-- Looking for clock_gettime in rt - found
-- Found PythonLibs: /usr/lib/arm-linux-gnueabihf/libpython3.4m.so (found version "3.4.2")
-- git found: libcec-2.1.4+223-2f62595
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.28")
-- Looking for include file sys/file.h
-- Looking for include file sys/file.h - found
-- Looking for flock
-- Looking for flock - found
-- checking for module 'udev'
--   found udev, version 215
-- Looking for 3 include files X11/Xlib.h, ..., X11/extensions/Xrandr.h
-- Looking for 3 include files X11/Xlib.h, ..., X11/extensions/Xrandr.h - found
-- Looking for XRRGetScreenResources in Xrandr
-- Looking for XRRGetScreenResources in Xrandr - found
-- Found SWIG: /usr/bin/swig2.0 (found version "2.0.12")
-- Configured features:
-- Pulse-Eight CEC Adapter:                yes
-- Pulse-Eight CEC Adapter detection:      yes
-- xrandr support:                         yes
-- Raspberry Pi support:                   no
-- TDA995x support:                        no
-- Exynos support:                         no
-- DRM support:                            yes
-- Python support:                         version 3.4.2 (3.4)
-- lib info: git revision: libcec-2.1.4+223-2f62595, compiled on Fri 11 Dec 19:04:21 UTC 2015 by pi@raspberrypi on Linux 4.1.7-v7+ (armv7l), features: P8_USB, DRM, P8_detect, randr
-- Configuring done
-- Generating done
-- Build files have been written to: /home/pi/libcec/build
```
I realize the last `cmake ..` command I sent didn't manually set the directories - I did this because having the paths declared didn't make a difference on my last attempt.  I've been reinstalling from the SD after each failed attempt, just to be sure that I didn't mess things up on each attempt.  I didn't want to go through the correct installation process after a bad attempt on the off chance that I could have screwed something up.

--- a/src/libcec/cmake/CheckPlatformSupport.cmake
+++ b/src/libcec/cmake/CheckPlatformSupport.cmake
@@ -93,8 +93,9 @@ else()
 
   # raspberry pi
   find_library(RPI_BCM_HOST bcm_host "${RPI_LIB_DIR}")
-  check_library_exists(bcm_host bcm_host_init "${RPI_LIB_DIR}" HAVE_RPI_API)
-  if (HAVE_RPI_API)
+  check_library_exists(bcm_host bcm_host_init "${RPI_LIB_DIR}" HAVE_RPI_LIB)
+  if (HAVE_RPI_LIB)
+    set(HAVE_RPI_API 1)
     find_library(RPI_VCOS vcos "${RPI_LIB_DIR}")
     find_library(RPI_VCHIQ_ARM vchiq_arm "${RPI_LIB_DIR}")
     include_directories(${RPI_INCLUDE_DIR} ${RPI_INCLUDE_DIR}/interface/vcos/pthreads ${RPI_INCLUDE_DIR}/interface/vmcs_host/linux)

